src/utils/undirected_graph.hpp

```diff
@@ -696,6 +697,52 @@ Weight undirected_graph<Vertex, Weight>::dijkstraTraversal(const Vertex& vertex,
     return total;
 }
 
+/**
+ * This function takes two verticies and returns all the edges between the
+ * two verticies and the total Weight of all the edges
+ *
+ * @param vertexFrom records all edges starting from this Vertex (origin)
+ * @param vertexTo records all edges between start and this vertex (destination)
+ */
+template<typename Vertex, typename Weight>
+std::pair<std::list<std::pair<Vertex,Vertex>>,Weight>
+undirected_graph<Vertex, Weight>::dijkstraTraversal(const Vertex& vertexFrom, const Vertex& vertexTo) const
+{
+    std::map<Vertex,std::pair<Weight,Vertex>> routes;
+    std::list<std::pair<Vertex,Vertex>> route;
+
+    if(vertexExists(vertexFrom) && vertexExists(vertexFrom))
```

Comment:
Whys does this line have a repetitive check?

Reply:
Oh nooooo

Reply:
My bad it's supposed to check if both to and from exist, I'll fix it